docs/third-party-developers/extensibility/data-store/payment.md

```diff
@@ -27,19 +27,19 @@ with. We do not encourage extensions to dispatch actions onto this data store ye
 
 ## Selectors
 
-### isPaymentPristine
+### isPaymentIdle
```

Comment:
Is it worth keeping `isPaymentPristine` in the docs, because it still exists? Maybe we should keep and describe it, but note that it has been deprecated?

Reply:
It doesn't exist anymore. I replace it with isPaymentIdle, but on second thought it would be better to deprecate it just in case so I'll do that